libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -98,6 +98,22 @@ void AP_Mount_Backend::control(int32_t pitch_or_lat, int32_t roll_or_lon, int32_
     }
 }
 
+// handle a GLOBAL_POSITION_INT message
+bool AP_Mount_Backend::handle_global_position_int(uint8_t msg_sysid, const mavlink_global_position_int_t &packet)
+{
+    if (_state._target_sysid != msg_sysid) {
```

Comment:
the target_sysid state could be moved out of `_state` and into the `protected:` section in the mount backend if we were suitably motivated.  doesn't need to be done as part of this PR.

Reply:
if you want me to do it I can, you just need to say a clear yes :)

Reply:
Not in this PR :-)

If you feel like doing it in a follow-up PR that would be cool.